data_steward/cdr_cleaner/cleaning_rules/unit_normalization.py

```diff
@@ -0,0 +1,302 @@
+        try:
+            bq.create_tables(client, self.get_project_id(),
+                             [unit_mapping_table])
+            # Uploading data to _unit_mapping table
+            unit_mappings_csv_path = os.path.join(resources.resource_files_path,
+                                                  UNIT_MAPPING_FILE)
+            result = bq.upload_csv_data_to_bq_table(
+                client, self.get_dataset_id(), UNIT_MAPPING_TABLE,
+                unit_mappings_csv_path, UNIT_MAPPING_TABLE_DISPOSITION)
+            LOGGER.info(
+                f"Created {self.get_dataset_id()}.{UNIT_MAPPING_TABLE} and loaded data from {unit_mappings_csv_path}"
+            )
+        except RuntimeError as err:
+            LOGGER.info(f"{unit_mapping_table} already exists.")
+            result = err
+        return result
+
+    def get_query_specs(self):
+        """
+        :return:
+        """
+        sandbox_query = dict()
+        sandbox_query[cdr_consts.QUERY] = SANDBOX_UNITS_QUERY.render(
+            project_id=self.get_project_id(),
+            sandbox_dataset=self.get_sandbox_dataset_id(),
+            intermediary_table=self.get_sandbox_tablenames()[0],
+            dataset_id=self.get_dataset_id(),
+            unit_table_name=UNIT_MAPPING_TABLE,
+            measurement_table=MEASUREMENT)
+
+        update_query = dict()
+        update_query[cdr_consts.QUERY] = UNIT_NORMALIZATION_QUERY.render(
+            project_id=self.get_project_id(),
+            dataset_id=self.get_dataset_id(),
+            unit_table_name=UNIT_MAPPING_TABLE,
+            measurement_table=MEASUREMENT)
+        update_query[cdr_consts.DESTINATION_TABLE] = MEASUREMENT
+        update_query[cdr_consts.DISPOSITION] = bq_consts.WRITE_TRUNCATE
+        update_query[cdr_consts.DESTINATION_DATASET] = self.get_dataset_id()
+        return [sandbox_query, update_query]
+
+    def setup_validation(self, client):
+        """
+        Run required steps for validation setup
+        """
+        pass
+
+    def validate_rule(self, client):
+        """
+        Validates the cleaning rule which deletes or updates the data from the tables
+        """
+        pass
+
+    def get_sandbox_tablenames(self):
+        sandbox_table_names = list()
+        sandbox_table_names.append(self._issue_numbers[0].lower() + '_' +
+                                   self._affected_tables[0])
+        return sandbox_table_names
+
+
+if __name__ == '__main__':
+    import cdr_cleaner.args_parser as parser
+    import cdr_cleaner.clean_cdr_engine as clean_engine
+
+    ARGS = parser.parse_args()
+    clean_engine.add_console_logging(ARGS.console_log)
+    unit_normalization = UnitNormalization(ARGS.project_id, ARGS.dataset_id,
+                                           ARGS.sandbox_dataset_id)
+    client = bq.get_client(ARGS.project_id)
+    UnitNormalization.setup_rule(unit_normalization, client=client)
+    query_list = UnitNormalization.get_query_specs(unit_normalization)
```

Comment:
You would call these methods from the `unit_normalization` instance you created.  Like:
```
# save the result in case you want to use it.  Otherwise, you can just call the method without assigning the return value to a variable
result = unit_normalization.setup_rule(client)
query_list = unit_normalization.get_query_specs()
```